### PR TITLE
fix: Update .travis.yml format

### PR DESCRIPTION
**Pull request by riddell-stan**

### CODE REVIEW

File: env_vars.sh [hunk 1]
```diff
@@ -1,5 +1,3 @@
 # See multibuild/README.rst
 export MACOSX_DEPLOYMENT_TARGET=10.9
 export CFLAGS="-static-libstdc++"
```
Comment (ahartikainen) [inline env_vars.sh:3]: Is this causing the problem that C++ exceptions are not catched?

File: .travis.yml [hunk 2]
```diff
@@ -1,109 +1,110 @@
...
-        - MB_PYTHON_VERSION=3.8
-        - NP_BUILD_DEP=1.14.5
+jobs:
+  - os: linux
+    env:
+      - MB_PYTHON_VERSION=2.7
+  - os: linux
+    env:
+      - MB_PYTHON_VERSION=2.7
+      - UNICODE_WIDTH=16
```
Comment (ahartikainen) [inline .travis.yml]: Latest manylinux2014 has dropped support for 2.7. Not sure about manylinux1.

And 2.7 is eol, and 2.19.1 wheel is in pypi.

**Review by ahartikainen — COMMENTED** [review]
C++ exception is not catched or something similar is going on. When inv file is not defined, it should just skip that part, now it send error to log and crashes in the return step.

**Comment by riddell-stan** [comment]
I'm fairly sure I screwed up and deleted some important patches I had to the default manylinux1 image. IIRC, unpatched multibuild does not build the linux wheels for pystan 2.19.1 due to the old version of gcc.

I'm really sorry about this! I might have time to look at this again after the next pystan 3 beta is out.

**Comment by ahartikainen** [comment]
These for some reason work

https://travis-ci.org/github/ahartikainen/pystan-wheels/builds/716910572
https://github.com/ahartikainen/pystan-wheels/pull/1/files

TBH not sure what all did I change, but I think I just manually forced `manylinux1` (multibuild goes for `manylinux2014`)

**Comment by ahartikainen** [comment]
I added explicit line for `manylinux1`, let's see if that works.

**Comment by ahartikainen** [comment]
Everything else but this works
```
- os: linux
    env:
      - MB_PYTHON_VERSION=2.7
      - UNICODE_WIDTH=16
```

Reason for this is that `pip` upgrade fails.

Should we merge this and then decide what to do for py27.

(This pr uploads only wheels for 38)

**Review by ahartikainen — APPROVED** [review]

File: .travis.yml (hunk 2 above)
Comment (riddell-stan) [reply .travis.yml]: Let's get rid of it then.

Comment (ahartikainen) [reply .travis.yml]: Do you want me to add a commit for it, or do you want to do it yourself?

**Comment by riddell-stan** [comment]
Removed all 2.7 versions and 3.5 too.

**Review by ahartikainen — APPROVED** [review]

**Pull request merged**